tests/php/src/PrivateAccess.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Trait PrivateAccess.
+ *
+ * @package Amp\AmpWP
+ */
+
+namespace Amp\AmpWP\Tests;
+
+use ReflectionClass;
+use ReflectionException;
+
+/**
+ * Trait PrivateAccess.
+ *
+ * Allows accessing private methods and properties for testing.
+ *
+ * @internal
+ * @since 1.5.0
+ */
+trait PrivateAccess {
```

Comment:
As this is something we need multiple times across multiple tests, I thought it made sense to abstract it away into a trait.

I started a new namespace for testing using Composer's `autoload-dev` directive, so it is always accessible without additional includes during tests.

Reply:
Great idea, no sense repeating this logic.

Reply:
Very nice.